### DISCUSSION TITLE
srsRAN Project Fall Workshop 2024

### DISCUSSION THREAD

**Discussion by brendan-mcauliffe** [srsran/srsRAN_Project#743] [Announcements]
![srsRan_workshop_V04_1](https://github.com/user-attachments/assets/e451d17f-960e-463f-98fb-5f59dd6dd5b3)

Registration is now open for the second srsRAN Project Workshop. Held in partnership with the Commonwealth Cyber Initiative (CCI), the workshop will take place on October 1 & 2, 2024 in Arlington, Virginia at the Virginia Tech Research Center. 

This workshop will bring together the community of srsRAN Project users and enthusiasts from across Academia, Industry and Government. Participants will hear from leading experts in the area of Open RAN, SDR, and Next Generation Wireless and from colleagues about their experiences of using srsRAN Project to further their R&D objectives. The workshop will include technical presentations and tutorials from the srsRAN Project developers as well as presentations, demonstrations and lightning talks from the broader community.

Workshop capacity is limited and registration is available on a first-come, first-served basis. Register [here](https://srsran-project-workshop-2024.eventbrite.com ). 

This year we are also accepting proposals for talks and demos. Talks will focus on work using srsRAN Project and should be up to one hour in length. Demos will be displayed for the entire two-day duration of the event. Each demo will be assigned a fixed location at the workshop venue. 

You can submit your proposal through [this form](https://forms.gle/8mnzUbdY8ckH4yYR8).

**Comment by brendan-mcauliffe** [comment]
Want to present your work at the srsRAN Project Fall Workshop in Virginia? Now is your chance. The call for presentations, demos, and talks is now open. Submit your proposal here: 

https://forms.gle/8mnzUbdY8ckH4yYR8 

The deadline for submissions is August 31st.

**Reply by Zplusless** [reply]
Is online participation supported?

**Reply by brendan-mcauliffe** [reply]
Hi @Zplusless, the workshop is in person only. Some of the material will be made available online after the workshop.

**Reply by Zplusless** [reply]
> Hi @Zplusless, the workshop is in person only. Some of the material will be made available online after the workshop.

Thanks！Wish the workshop great success.
